code/modules/mob/living/silicon/robot/death.dm

```diff
@@ -26,6 +26,8 @@
 
 	. = ..()
 
+	remove_traits(list(TRAIT_HANDS_BLOCKED, TRAIT_INCAPACITATED, TRAIT_FLOORED, TRAIT_CRITICAL_CONDITION), STAT_TRAIT) // to remove unwanted traits from living/set_stat
```

Comment:
When do these traits get added and why do we have to snowflake remove them here?

Reply:
These traits are added by [living/set_stat](https://github.com/yogstation13/Yogstation/blob/2a40a7701fe12bdff80dda210ba6d5e7a67d5fe6/code/modules/mob/living/living.dm#L1489). It is called [here](https://github.com/yogstation13/Yogstation/blob/2a40a7701fe12bdff80dda210ba6d5e7a67d5fe6/code/modules/mob/living/death.dm#L52) which was added by the [antag pr](https://github.com/yogstation13/Yogstation/pull/18514). It worked and I couldn't think of a better solution at a time.

Reply:
We have a trait that is meant to block this, ``TRAIT_FORCED_STANDING``. It hasn't been implemented properly (likely due to the scope of the PR) so currently does nothing and is the reason why this is a problem.